app/controllers/application_controller.rb

```diff
@@ -7,6 +7,7 @@ class ApplicationController < ActionController::Base
   include ApplicationMultitenancyConcern
   include ApplicationInternationalizationConcern
   include ApplicationThemingConcern
+  include ApplicationDeviseConcern
```

Comment:
I put it separately because this is devise specific, and I think we will have a serious devise configurations in future ?